composables/masto/publish.ts

```diff
@@ -21,6 +21,17 @@ export function usePublish(options: {
   const isExpanded = $ref(false)
   const failedMessages = $ref<string[]>([])
 
+  const publishSpoilerText = $computed({
+    get() {
+      return draft.params.sensitive ? draft.params.spoilerText : ''
+    },
+    set(val) {
+      if (!draft.params.sensitive)
+        return
```

Comment:
Do we need this?

```suggestion
```

Reply:
No, just to make the logic more rigorous, is it recommended to delete it?

Reply:
I can't really think when this would be called, so I think it probably doesn't matter either way.

Reply:
@danielroe  Got it, Normal operation here would never call. I wish I could remove this if you prefer cleaner code.